lib/lhm/migrator.rb

```diff
@@ -205,5 +206,11 @@ def index_ddl(cols, unique = nil, index_name = nil)
     def order_column
       @options[:order_column] || @origin.pk
     end
+
+    def assert_valid_idx_name(index_name)
+      if index_name && !(index_name.is_a?(String) || index_name.is_a?(Symbol))
+        raise ArgumentError, "index_name must be a string or symbol"
```

Comment:
> you have to wait after deploying to realize shit's broken

Hum, wat? If the migration fails then it cannot be deployed. Shouldn't even pass CI.

On the other hand the exception I'm adding above is preventing a valid but bad migration from going out. If the index is too long then the LHM should blow up at run time, which is good enough IMHO.